redmine/src/main/java/org/arquillian/extension/governor/redmine/api/IssueStatus.java

```diff
@@ -0,0 +1,25 @@
+package org.arquillian.extension.governor.redmine.api;
+
+/**
+ * Created by pestano on 20/12/15.
```

Comment:
please use this url as author tag <a href="mailto:rmpestano@gmail.com">Rafael Pestano</a>